src/index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-    <body>
+    <body style="#00a2e8; color:white; text-align:center;">
```

Comment:
Thanks for updating the README! 

I see one last issue with this styling. You'll need to be sure to add `background-color` to the style. You added the color itself but missed the attribute name. Try this out:

`<body style="background-color: #00a2e8; color:white; text-align:center;">`